tests/test_data/valid/pixels/pixel_guide.json5

```diff
@@ -61,6 +61,34 @@
                 },
                 // Examples - used when params are more open-ended
                 "examples": [["tracker1", "tracker2"], ["tracker3"]]
+            },
+            {
+                "key": "customMetadata",
```

Comment:
Will update the README as well after we run through stakeholder feedback (not expecting much pushback on this)